extensions/clearinghouse/src/main/java/de/sovity/extension/clearinghouse/ClearingHouseExtension.java

```diff
@@ -155,7 +155,7 @@ private void registerCommonTypes(TypeManager typeManager) {
     }
 
     private void registerClearingHouseMessageSenders(ServiceExtensionContext context) {
-        var httpClient = context.getService(OkHttpClient.class);
+        var httpClient = context.getService(EdcHttpClient.class);
```

Comment:
Can be removed as it is not used anymore

Reply:
Needed for `IdsMultipartSender`?!